server/integration_tests/test_data/demo_fallback/query_1/chart_config.json

```diff
@@ -1,116 +1,15 @@
 {
   "client": "test",
-  "config": {
-    "categories": [
-      {
-        "blocks": [
-          {
-            "columns": [
-              {
-                "tiles": [
-                  {
-                    "type": "PLACE_OVERVIEW"
-                  }
-                ]
-              }
-            ]
-          }
-        ]
-      }
-    ],
-    "metadata": {
-      "placeDcid": [
-        "geoId/0667000"
-      ]
-    }
-  },
+  "config": {},
   "context": {},
   "debug": {},
-  "entities": [],
-  "pastSourceContext": "",
+  "failure": "Sorry, could not complete your request. No place found in the query.",
   "place": {
-    "dcid": "geoId/0667000",
```

Comment:
This is very odd -- place detection on autopush for this query seems to work:

https://autopush.datacommons.org/explore#q=Number%20of%20Shakespeare%20fans%20in%20San%20Francisco%20and%20Chicago